addons/framework/statsTracking/fn_entityKilled.sqf

```diff
@@ -18,27 +18,27 @@ switch true do
 {
 	//If a player kills a CIV
 	case (isPlayer _instigator && !(_sideKilled in ["potato_w", "potato_i", "potato_e"])):{
-		[1, 8, (getPlayerUID _instigator)] call CMF_fnc_updateStatArray;
+		[1, 8, (_instigator getVariable "Player_UID")] call CMF_fnc_updateStatArray;
```

Comment:
Why this change? `getPlayerUID _instigator` is much faster than getting a variable.

Reply:
If a player dies, then their UID is no longer linked to that body. So we have to use a variable to identify the dead players UID.